modules/apps/foundation/contacts/contacts-web/src/main/java/com/liferay/contacts/web/upgrade/ContactsWebUpgrade.java

```diff
@@ -28,6 +29,10 @@
 	@Override
 	public void register(Registry registry) {
 		registry.register(
+			"com.liferay.contacts.web", "0.0.0", "1.0.0",
+			new DummyUpgradeStep());
```

Comment:
@migue @csierra I'm not familiar enough with the new upgrade fw to judge this. Does it look ok to you?

Drew, if it looks good to them, go ahead and send it to Brian.